Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ghcr.io/shyim/wolfi-php/frankenphp:8.3 AS base
+RUN apk add --no-cache \
+    php-frankenphp-8.3-opcache \
+    php-frankenphp-8.3-mbstring \
+    php-frankenphp-8.3-intl \
+    php-frankenphp-8.3-dom \
+    php-frankenphp-8.3-curl
+
+FROM base AS prod
+COPY . .
+
+FROM composer/composer:latest-bin AS composer
+
+FROM base AS dev
+RUN apk add --no-cache \
```

Comment:
I'd add `curl`, `wget`, `git`, `htop` if there are not such

Reply:
To prod as well

Reply:
Why do we need these in prod?